redaxo/src/addons/install/pages/packages.update.php

```diff
@@ -52,6 +52,11 @@
 } elseif ($addonkey && isset($addons[$addonkey])) {
     $addon = $addons[$addonkey];
 
+    $version = rex_escape(rex_addon::get($addonkey)->getVersion());
+    if (rex_version::isUnstable($version)) {
```

Comment:
Der Installer läuft so nur noch mit 5.10.

Ich hatte allerdings eigentlich vor, den Installer separat auch als Beta anzubieten, und dann aufzufordern, dass manche erst Installer updaten, und dann den Core.
Um das Core-Update auch zu testen, da es da ja Änderungen gab im Installer (die sonst erst beim nächsten Update greifen).

Schlage daher das vor:

```suggestion
    if (class_exists(rex_version::class) && rex_version::isUnstable($version)) {
```